### PR TITLE
Improve host aliases page

### PR DESCRIPTION
**Pull request by kairen**
Fixes issue #5442. Add more detail command, and fix confusion code block.

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/5443)

### CODE REVIEW

**Comment by kairen** [comment]
@xiangpengzhao Could you help me to review this changes?

**Comment by xiangpengzhao** [comment]
ping @rickypai who is the author of this doc for reviewing :)

File: docs/concepts/services-networking/add-entries-to-pod-etc-hosts-with-host-aliases.md [hunk 1]
```diff
@@ -42,8 +47,20 @@ In addition to the default boilerplate, we can add additional entries to the hos
...
 {% include code.html language="yaml" file="hostaliases-pod.yaml" ghlink="/docs/concepts/services-networking/hostaliases-pod.yaml" %}
 
-The hosts file content would look like this:
+This Pod can be started with the following commands:
+
+```shell
+$ kubectl apply -f hostaliases-pod.yaml
+pod "hostaliases-pod" created
+
+$ kubectl get pod -a
```
Comment (rickypai) [inline docs/concepts/services-networking/add-entries-to-pod-etc-hosts-with-host-aliases.md]: can you do `-o=wide` here so it prints out the pod IP. that way the IP of this entry

```
10.244.3.80	hostaliases-pod
```

becomes clearer

**Comment by rickypai** [comment]
thanks for fixing this bte

File: docs/concepts/services-networking/add-entries-to-pod-etc-hosts-with-host-aliases.md (hunk 1 above)
Comment (kairen) [reply docs/concepts/services-networking/add-entries-to-pod-etc-hosts-with-host-aliases.md]: @rickypai Okay!

**Comment by kairen** [comment]
@xiangpengzhao Thanks you for your help!!

**Comment by kairen** [comment]
@rickypai I added wide output, thanks you for review.

**Comment by chenopis** [comment]
I think this is good to go.

**Pull request merged**